modules/account-lib/src/coin/hbar/utils.ts

```diff
@@ -305,6 +304,7 @@ export function isValidAddressWithPaymentId(address: string): boolean {
 }
 
 /**
+<<<<<<< HEAD
```

Comment:
nit: Old conflict comment to remove

Reply:
gonna have another PR after this, so will remove it :D